source/MaterialXRender/Handlers/GeometryHandler.h

```diff
@@ -1,124 +1,95 @@
 #ifndef MATERIALX_GEOMETRYHANDLER_H
 #define MATERIALX_GEOMETRYHANDLER_H
 
-#include <MaterialXCore/Types.h>
-#include <string>
+#include <MaterialXRender/Handlers/Mesh.h>
 #include <memory>
-#include <vector>
+#include <map>
 
 namespace MaterialX
 {
+
+/// Shared pointer to a GeometryLoader
+using GeometryLoaderPtr = std::shared_ptr<class GeometryLoader>;
+
+/// @class @GeometryLoader
+/// Base class representing a geometry loader. A loader can be 
+/// associated with one or more file extensions.
+class GeometryLoader
+{
+  public:
+    /// Default constructor
+    GeometryLoader() {}
+
+    /// Default destructor
+    virtual ~GeometryLoader() {}
+
+    /// Returns a list of supported extensions
+    /// @return List of support extensions
+    const std::vector<std::string>& supportedExtensions()
```

Comment:
Code convention: we normally skip the std qualifier on strings and vectors, as they are not needed. For longer term it's then also easier to introduce another implementation for strings or vectors, if needed.

For this case there is also a type alias to use directly: StringVector

Reply:
Ah yes. I'll fix this